test/system/user_test.rb

```diff
@@ -0,0 +1,103 @@
+require "application_system_test_case"
+
+class UsersTest < ApplicationSystemTestCase
```

Comment:
Please make sure the file and class names match (the class is plural).

Reply:
Whoops! Renamed.